pkg/clustermanager/internal/deployments.go

```diff
@@ -16,23 +13,6 @@ var ExternalEtcdDeployments = map[string][]string{
 	"etcdadm-bootstrap-provider-system": {"etcdadm-bootstrap-provider-controller-manager"},
 }
 
-// map between file name and the capi/v deployments
-var ClusterDeployments = map[string]*types.Deployment{
```

Comment:
I was going to remove the webhook pods from this, but it shows up as an unused var so thought I'll remove it